product_import/wizard/product_import.py

```diff
@@ -215,7 +231,7 @@ def import_button(self):
             raise UserError(_("This catalogue doesn't have any product!"))
         company_id = self._get_company_id(catalogue)
         seller = self._get_seller(catalogue)
-        self.with_context(company_id=company_id)._create_products(
+        self.with_context(product_company=company_id)._create_products(
```

Comment:
```suggestion
        self.with_context(product_company_id=company_id)._create_products(
```
to be more consistent